training/train.py

```diff
@@ -141,7 +140,8 @@ def validate_manifest_data(manifest_list):
 
 def preprocess_raw_data(raw_data_dict, lower_limit, upper_limit):
     """Preprocess raw data."""
-    all_manifest_list = raw_data_dict.get('package_list', [])
+    all_manifest_list = raw_data_dict.get('user_input_stack', []) \
```

Comment:
I assume we're filtering for duplicates somewhere right? Otherwise you can do set(package_list) union set(user_input_stack).

Reply:
Yes we are removing duplicates in https://github.com/fabric8-analytics/f8a-pypi-insights/blob/c147fda216ec1ea22d62b4dca664ec655689257a/training/train.py#L114.